evb/evb_input.c

```diff
@@ -25,25 +25,25 @@
  * .. flat-table:: Button map
  *     :widths: 1 1
  *     :header-rows: 1
- * 
+ *
  *     * - Button/Direction
  *       - Linux key code (value)
- * 
+ *
  *     * - Back button press
  *       - ``KEY_BACKSPACE`` (14)
- * 
+ *
  *     * - Joystick button press
  *       - ``KEY_ENTER`` (28)
- * 
+ *
  *     * - Joystic up
```

Comment:
Look like we can fix the spelling of joystic here too. And several repeats below.